http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -185,11 +186,20 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
                                                          false
       )
       aksSubnetName <- getLandingZoneResourceName(groupedLzResources, "DeployedSubnet", AKS_NODE_POOL_SUBNET, false)
-      postgresName <- getLandingZoneResourceName(groupedLzResources,
+      postgresResource <- getLandingZoneResource(groupedLzResources,
                                                  "Microsoft.DBforPostgreSQL/flexibleServers",
-                                                 SHARED_RESOURCE,
-                                                 false
+                                                 SHARED_RESOURCE
       ).attempt
```

Comment:
The `.attempt` here is tricky -- maybe a comment would be good that it's there to allow for older landing zones which may not have a Postgres Flex deployed.